pyscriptjs/package-lock.json

```diff
@@ -1,93 +1,501 @@
 {
   "name": "pyscript",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Hi, IMHO you should update your npm.

https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json#lockfileversion

Reply:
+1. If you have `nvm` (Node Version Manager) installed, you might be using something like Node v14 which uses an older version of NPM. Try `nvm use v16` to switch version for the terminal session, or `nvm alias default v16` to set it as your default Node version

Reply:
+1. If you have `nvm` (Node Version Manager) installed, you might be using something like Node v14 which uses an older version of NPM. Try `nvm use v16` to switch version for the terminal session, or `nvm alias default v16` to set it as your default Node version